src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services10.php

```diff
@@ -80,10 +80,12 @@ protected function getTestService()
      */
     public function getParameter($name)
     {
-        $name = strtolower($name);
+        if (!(isset($this->parameters[$name]) || isset($this->loadedDynamicParameters[$name]) || array_key_exists($name, $this->parameters))) {
+            $name = strtolower($name);
```

Comment:
should we deprecate the case insensitivity of parameter names ? This would be consistent with service ids.